docs/api/source/index.rst

```diff
@@ -1,12 +1,79 @@
 :orphan:
 
-================
-solid-client API
-================
+@inrupt/solid-client API Documentation
+======================================
 
+`@inrupt/solid-client <https://docs.inrupt.com/developer-tools/javascript/client-libraries/>`__
```

Comment:
We may wish to change this URL to point to either https://www.npmjs.com/package/@inrupt/solid-client or the GitHub repo.

Reply:
I think we should point to the npm package.
Also, we may want to standardise our READMEs a bit more.
Some simple improvements include badges pointing to the npm package and showing tests passing.

Reply:
I've pointed to the npm package, but the badges and standardisation is totally out of scope here.